mars/custom_bash.cases

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please rename the file to bash_rm_1356213.cases

Reply:
Done. Please check it.